Mathlib/GroupTheory/Submonoid/Operations.lean

```diff
@@ -121,42 +128,51 @@ section
 variable {A : Type _} [AddZeroClass A]
 
 /-- Additive submonoids of an additive monoid `A` are isomorphic to
-multiplicative submonoids of `multiplicative A`. -/
+multiplicative submonoids of `Multiplicative A`. -/
 @[simps]
-def AddSubmonoid.toSubmonoid : AddSubmonoid A ≃o Submonoid (Multiplicative A)
-    where
+def AddSubmonoid.toSubmonoid :
+    AddSubmonoid A ≃o
+      Submonoid
+        (Multiplicative
+          A) where
```

Comment:
This had been
```suggestion
def AddSubmonoid.toSubmonoid : AddSubmonoid A ≃o Submonoid (Multiplicative A)
    where
```
as outputted by mathport, did that not parse?  I think it's easier on the eye.